src/chargen.cpp

```diff
@@ -1084,6 +1127,37 @@ static void start_game(descriptor_data *d, const char *origin)
     GET_LOADROOM(d->character) = RM_CHARGEN_START_ROOM;
   }
 
+  // Otaku get some starting gear, so assign that here
+  if (d->ccr.is_otaku) {
+    // Equip cyberware (deduct essence and modify stats as appropriate)
+    obj_data *temp_obj;
+    for (int cyb = 0; cyb < OTAKU_CYBERWARE_NUM; cyb++) {
+      if (otaku_cyberware[cyb]) {
+        if (!(temp_obj = read_object(otaku_cyberware[cyb], VIRTUAL, OBJ_LOAD_REASON_ARCHETYPE))) {
+          mudlog_vfprintf(CH, LOG_SYSLOG, "SYSERR: Invalid cyberware item %ld specified for otaku class.",
+            otaku_cyberware[cyb]);
+          mudlog(buf, CH, LOG_SYSLOG, TRUE);

```

Comment:
Is there something specific in `buf` you're logging here, or is this a remnant of the old logging line?

Reply:
Artifact. Fixed.